packages/react/src/Card/CardActions.tsx

```diff
@@ -0,0 +1,72 @@
+import { forwardRef, ReactNode, CSSProperties } from 'react';
+import { cardClasses as classes } from '@mezzanine-ui/core/card';
+import { cx } from '../utils/cx';
+import { NativeElementPropsWithoutKeyAndRef } from '../utils/jsx-types';
+import { ButtonProps } from '../Button';
+import ConfirmActions from '../ConfirmActions';
+
+export interface CardActionsProps extends NativeElementPropsWithoutKeyAndRef<'div'> {
+  /**
+   * Content of cancel button. not render if value is empty.
+   */
+  cancelText?: string,
+  /**
+   * Content of confirm button. not render if value is empty.
+   */
+  confirmText?: string,
+  /**
+   * Click handler for cancel button.
+   */
+  onCancel?: ButtonProps['onClick'];
+  /**
+   * Click handler for confirm button.
+   */
+  onConfirm?: ButtonProps['onClick'];
+  /**
+   * The action bottom on the left. not render if value is empty.
+   */
+  otherActions?: ReactNode;
+  /**
+   * Inline style to apply to the card actions.
+   */
+  style?: CSSProperties;
+}
+/**
+ * The react component for `mezzanine` cardActions.
+ */
+const CardActions = forwardRef<HTMLDivElement, CardActionsProps>(function CardActions(props, ref) {
+  const {
+    cancelText,
+    className,
+    confirmText,
+    onCancel,
+    onConfirm,
+    otherActions,
+    style,
+    ...rest
+  } = props;
+
+  return (
+    otherActions || confirmText || cancelText ? (
```

Comment:
Unnecessary condition. Users should know what elements should be shown in CardActions component. Splitting otherActions display logic and confirmActions display logic is probably a better approach.